### DISCUSSION TITLE
xrdp can't connect some user? the window will crash

### DISCUSSION THREAD

**Discussion by XR-stb** [neutrinolabs/xrdp#2753] [Q&A]
here is my question. you can be no care.
And more bum, my ubuntu has three user: xiaoran, root, test1, i can use xrdp to connect root and test1, but when i connect xiaoran, the window will crash（even i had log out the user : xiaoran）, i don know how to see the log, if someone can tell me, that's very thanks
 
your can see my problem in: [https://youtu.be/Ruy6W2FKuPQ](https://youtu.be/Ruy6W2FKuPQ)

**Comment by XR-stb** [comment]
and my os is ubuntu 18.04, xrdp is 3.0

**Reply by metalefty** [reply]
That's strange. We never released xrdp 3.0.

**Reply by XR-stb** [reply]
oh， sorry， my xrdp is 0.9.12, i forgot it
